### DISCUSSION TITLE
CONUS404: Precip overestimation over urban areas

### DISCUSSION THREAD

**Discussion by mirizarry-ortiz** [hytest-org/hytest#402] [Dataset Questions]
From Figure 4 of the CONUS404 publication (Rasmussen et al., 2023) it is evident that CONUS404 is overestimating convective precipitation over urban centers. Any idea why that might be and what model parameters (if any) could be tweaked to avoid that? We know that PRISM might not be perfect but, at least for Florida, the average annual precipitation of 100 inches/year simulated by CONUS404 over the Miami-WPB urban areas is way too high. Precipitation is overestimated over other urban areas in FL as well, but not as drastically as over Miami-WPB.

There are studies that show that the UHI enhances wet season precipitation over urban centers and areas downwind. A study by Marshall and others (2004) indicates that "increased sea-breeze convergence and small increases in convective precipitation over urban areas along the eastern coastal ridge of south Florida [Miami-WPB area] are caused by increased sensible heat flux, which appears related to urbanization during the period 1900-93." I understand this, but it seems like this effect might be overestimated in CONUS404.

I have also noticed that ERA5 wind speeds at 10 m have a low bias and this is translating into a low bias in CONUS404 10-m wind speeds over Florida (CONUS404 nudges to ERA5 winds above the PBL). A low bias in ERA5 wind speeds is very well known (at least over the oceans). Could the lower surface wind speeds be affecting precipitation over urban areas?

Thanks for any insights on the above.

**Comment by AndreasPrein** [comment]
Dear Michelle,
We noticed this overestimation as well and it is indeed related to a too-strong urban heat island. CONUS404 assumes 100 % sealed surfaces over urban grid cells. This increases the sensible heat flux over urban areas, which creates convergence. This convergence triggers thunderstorms that cause the wet bias over cities in tropical and sub-tropical regions during summer (we do not see this effect strongly in NYC for instance). 

We know how to improve the representation of urban areas in our simulation and implement them into simulations that focus on the Delaware River basin. We plan to implement these changes into future CONUS simulations. 

I also found an underestimation of 10 m wind in the CONUS404 simulations (e.g., see Fig. 2 in the supplement of [this paper](https://www.nature.com/articles/s41558-023-01852-9)). However, I am not sure how strong this is related to biases in ERA5.

Please let me know if you have any additional questions.
I would be eager to hear more about your experience with the CONUS404 simulation.

All the best,
Andreas

**Reply by mirizarry-ortiz** [reply]
Hi Andreas! Thanks so much for your response. We are developing a WRF model for the state of Florida and if you don't mind, we would like to get a better idea on what parameters you plan to tweak for the DRB to help with the urban heat island overestimation issue. I have done some validation of some CONUS404 precip and wind speed statistics over Florida and I am happy to share those preliminary results with you but just not in a public forum due to USGS restrictions with sharing unpublished analyses. Do you mind if I Email them to you directly? No pressure to respond to my Email in any timeframe.
Thanks so much, Michelle

**Reply by AndreasPrein** [reply]
Dear Michelle,
We are using the Noah-MP land surface model in our simulations. By default, Noah-MP uses sfurbanphysics = 0, which we changed to sfurbanphysics = 1 in the DRB simulations. This uses the 1D urban model and allows you to set an urban fraction for city areas and you can set urban irrigation (i.g., watering of grass areas in the city). This is still a quite simple model but helped us to improve the urban heat island in Delaware significantly.

It would be great if you could share your results with me. My email is prein@ucar.edu

Best,
Andreas

**Reply by jbellino-usgs** [reply]
Thank you for the information, Andreas. I'll incorporate that change into our model and see if we can get good results with it. Would you be able to share your values for the various `num_urban_*` and `irr_*` options that go along with the 1D UCM?

Best regards,
Jason

**Reply by mirizarry-ortiz** [reply]
Thanks so much for sharing your changes with us. I will Email you some results.

Michelle
